fortio_main.go

```diff
@@ -141,6 +141,8 @@ var (
 
 	maxStreamsFlag = flag.Uint("grpc-max-streams", 0,
 		"MaxConcurrentStreams for the grpc server. Default (0) is to leave the option unset.")
+
+	payloadSizeFlag = flag.Int("payload-size", fgrpc.DefaultPayloadSize, "Payload size to create random payload")
```

Comment:
you didn't add "replaces -payload when set" and didn't update README.md to match output of release/updateFlags.sh

let's make the help string
"Additional random payload size, replaces -payload when set > 0, must be smaller than -maxpayloadsizekb"

note: it may be more interesting for users to automatically set maxpayloadsize